contract/data/mail_message_subtype.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" ?>
+<odoo noupdate="1">
+    <record id="mail_message_subtype_invoice_created" model="mail.message.subtype">
+        <field name="name">Invoice created</field>
+        <field name="res_model">contract.contract</field>
+    </record>
```

Comment:
Why don't you specify here in the 'parent_id' field the existing sup-type 'Invoice created' (https://github.com/odoo/odoo/blob/14.0/addons/account/data/account_data.xml#L110)  so that each partner that has that subtype is automatically notified of the creation of an invoice from the contract ?

Reply:
Although it's a good idea IMO isn't possible because need to related a contract field but in this case don't exists these field and would be one2many field.